src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Metadata.cs

```diff
@@ -194,7 +194,7 @@ private static Checksum GetMetadataChecksumSlow(Solution solution, PortableExecu
                 solution,
                 checksum,
                 loadOnly,
-                createAsync: () => CreateMetadataSymbolTreeInfoAsync(solution, checksum, reference, cancellationToken),
```

Comment:
💡 Never remove cancellation tokens when fixing this diagnostic. Instead, use the following if the token is not currently used:

```
_ = cancellationToken;
```

Reply:
Filed https://github.com/dotnet/roslyn/issues/43408 to investigate MetadataInfoCreator.

Reply:
This one has still not been resolved. #43408 relates to _using_ the cancellation token. Use of the cancellation token is optional, but passing the cancellation token in the callback is not.

Reply:
I don't understand the need for passing a cancellation token to a method that we have already identified as non-cancellable, just to then suppress the unused parameter diagnostic for the token parameter. The token can be added back when we make it a cancellable operation.

Reply:
Asynchronous operations without a cancellation token parameter are a code smell. Leaving the code in this state runs the risk that someone adds the parameter in the future and incorrectly hooks it up. If the operation cannot be cancelled here, it's better to pass `CancellationToken.None` explicitly.